play.pokemonshowdown.com/testclient-beta.html

```diff
@@ -102,7 +102,7 @@ <h3><button class="closebutton" tabindex="-1" aria-label="Close"><i class="fa fa
 	<script nomodule src="/js/lib/ps-polyfill.js"></script>
 	<script src="../config/testclient-key.js"></script>
 	<script src="js/client-core.js" onerror="alert('You must build the client with `node build` before using testclient.html')"></script>
-
+	<script src="js/battledata.js" onerror="alert('You must build the client with `node build` before using testclient.html')"></script>
```

Comment:
```suggestion
	<script src="js/battledata.js"></script>
```
You only need one alert, alerting multiple times in a row is just annoying.

Reply:
Also what? Why? I thought we just include battledata's contents separately.

Reply:
to include the `BattleTextAFD` and `BattleTextNotAFD ` values from `text-afd.js`.
i saw it included `battledata.js` instead of `text-afd.js` in index.html, testclient.html so i thought i should to do the same in testclient-beta.html

Reply:
Ah yeah, the Preact client splits apart battledata so that some of it can be loaded earlier and some of it can be loaded later.

Reply:
battledata is a bundle of all of battle's dependencies:

```
'play.pokemonshowdown.com/src/battle-dex.ts',
'play.pokemonshowdown.com/src/battle-dex-data.ts',
'play.pokemonshowdown.com/src/battle-log.ts',
'play.pokemonshowdown.com/src/battle-log-misc.js',
'caches/pokemon-showdown/server/chat-formatter.ts',
'play.pokemonshowdown.com/data/text.js',
'play.pokemonshowdown.com/data/text-afd.js',
'play.pokemonshowdown.com/src/battle-text-parser.ts',
```

Preact splits it up because it needs `battle-dex` extremely early (for `toID`) and the others relatively late.

Reply:
> Ah yeah, the Preact client splits apart battledata so that some of it can be loaded earlier and some of it can be loaded later.

oh then its fine to just load the `text-afd.js` not `battledata.js` file. 
idk, actually i feel like we are gonna need this import anyway